sw/airborne/math/ActiveSetCtlAlloc/src/lib/qr_solve/qr_solve.c

```diff
@@ -0,0 +1,1622 @@
+/*
+ * This is part of the qr_solve library from John Burkardt.
+ * http://people.sc.fsu.edu/~jburkardt/c_src/qr_solve/qr_solve.html
+ *
+ * It is slightly modified to make it compile on simple microprocessors,
```

Comment:
is this solver really more efficient than the SVD-based solver already in the math lib `math/pprz_matrix_decomp_float.h` ?